packages/supermassive/src/codegen/context/expect.ts

```diff
@@ -0,0 +1,53 @@
+// import {
```

Comment:
Do we want to remove this ? or implement it @freiksenet

Reply:
Let's implement this later.